bedrock/settings/base.py

```diff
@@ -1069,3 +1075,51 @@ def lazy_email_backend():
     'dsn': config('SENTRY_DSN', None),
     'release': config('GIT_SHA', None),
 }
+
+# Django-CSP
+CSP_DEFAULT_SRC = (
+    "'self'",
+    '*.mozilla.net',
+    '*.mozilla.org',
+)
+CSP_IMG_SRC = (
+    "'self'",
+    '*.mozilla.net',
+    '*.mozilla.org',
+    '*.optimizely.com',
+    'www.googletagmanager.com',
+    'www.google-analytics.com',
+    '*.tiles.mapbox.com',
+    'api.mapbox.com',
+)
+CSP_SCRIPT_SRC = (
+    "'self'",
+    "'unsafe-inline'",
+    '*.mozilla.org',
+    '*.mozilla.net',
+    '*.optimizely.com',
+    'optimizely.s3.amazonaws.com',
+    'www.googletagmanager.com',
+    'www.google-analytics.com',
+)
+CSP_STYLE_SRC = (
+    "'self'",
+    "'unsafe-inline'",
+    '*.mozilla.org',
+    '*.mozilla.net',
+)
+CSP_CHILD_SRC = (
+    '*.optimizely.com',
+    'www.googletagmanager.com',
+    'www.google-analytics.com',
+)
+CSP_CONNECT_SRC = (
+    '*.optimizely.com',
+    'www.googletagmanager.com',
+    'www.google-analytics.com',
+    '*.tiles.mapbox.com',
+    'api.mapbox.com',
+)
+CSP_REPORT_ONLY = config('CSP_REPORT_ONLY', default=False, cast=bool)
+if config('CSP_REPORT_ENABLE', default=True, cast=bool):
```

Comment:
@glogiotatidis I think this is a good idea for lumbergh as well. It could be quite a lot of requests to the report URL if we make a mistake on a popular page.